README.md

```diff
@@ -125,6 +125,9 @@ assertRecyclerViewItemCount(R.id.recycler, 10);
 
 // And another tricky feature
 assertThatBackButtonClosesTheApp();
+
+// Check ImageView's drawable
```

Comment:
Let's match previous comments' style. What about "Is this ImageView showing this drawable?"